Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	
+++ b/main.py	(date 1675798909326)
@@ -5,7 +5,7 @@
 k = int (k)
 a = k//n
 b = k%n
-print("У кожного школяра буде по", a, "Яблук"\n"У кошику залишиться", b, "яблук")
+print("У кожного школяра буде по", a, "Яблук")
 print("У кошику залишиться", b, "яблук")
 
 
